panther_analysis_tool/rule.py

```diff
@@ -289,55 +379,84 @@ def run(
             return rule_result
 
         try:
-            rule_result.title_output = self._get_title(event, use_default_on_exception=batch_mode)
+            rule_result.title_defined = self._is_function_defined("title")
```

Comment:
Since the `_get_<auxiliary_function_result>` methods were returning `None`, it was not feasible to discern whether a `None` value was returned (acceptable return type for `destinations`) or the function was not defined. Now we know whether the function was defined, so we can display the output as well.